src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
 body {
   margin: 0;
 }
+
+.content {
+  margin: 0 auto;
+  width: 700px;
+  padding-top: 30px;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    &-image {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    width: 100%;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    letter-spacing: 0;
+    text-align: left;
+
+    &--position {
+      margin-top: 40px;
+    }
+  }
+
+  &__code {
+    width: 100%;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
ok